pubspec.yaml

```
@@ -4,11 +4,12 @@ version: 1.3.2
 homepage: https://github.com/xantiagoma/dart_date
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=3.0.2 <4.0.0"
+  flutter: ">=1.17.0"
 
 dependencies:
-  intl: ">=0.17.0 <0.19.0"
-  timeago: ^3.0.2
+  intl: ^0.19.0
```

```suggestion
  intl: ">=0.17.0 <0.20.0"
```

No need to artifically restrain this anymore

---

Why is a top restrain (<0.20.0) is needed?

---

Your `^0.19.0` actually includes also a top restrain. It is short for `>=0.19.0 <0.20.0`
However, since there are really no breaking changes in this case, we can lower the constraints to `>=0.17.0 <0.20.0`